### ISSUE TITLE
Monolith DKP - Management Issue

### ISSUE THREAD

**Issue by Krispxxy** [Roeshambo/MonolithDKP#20]
Hey!
Your addon is amazing and thank you so much for sharing it. I have commented on your updates youtube video with the same issue. I was messing around with the white list, accidentally added myself (GM) to whitelist cause I couldnt see the options tab. Wasnt really familiar with what it did. Updated the Addon, logged back in and now I cant manage anything within the addon but can see the options tab haha. Is there a way I can take myself off the whitelist or completely reset the addon? I have no DKP amounts. 

Thanks

**Comment by Roeshambo** [comment]
Were you able to get this resolved by deleting the savedvariables file? For future reference, if you don't want to delete the whole file and lose any records you may have had, you can go into the file with a text editor and remove the whole table labled MonDKP_Whitelist {} and all of it's contents.

**Comment by Krispxxy** [comment]
Yes, I was able to figure it out. I didn’t update the add-on before I started messing around with it and didn’t fully understand what the whitelist meant, so I put myself (GM) on the whitelist and it messed everything up. Your instructions fixed everything and I was able to reset the add-on. I will stay in contact you for future references. Thank you so much for the add-on. I was dreading manually imputing DKP on spread sheets for my guild. We are going to start raiding MC in about 3-4 weeks, so I will stay in contact if I run I have any questions!!

Thank you so much for the add-on!
Parker Brubaker

(602) 622 - 0396
